app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentPresenter.kt

```diff
@@ -126,6 +126,10 @@ class ResumeLessonFragmentPresenter @Inject constructor(
       resumeLessonViewModel.chapterSummary.get()!!.summary,
       binding.resumeLessonChapterDescriptionTextView
     )
+    if (binding.resumeLessonChapterDescriptionTextView.text.toString().isBlank()) {
+      binding.resumeLessonChapterDescriptionTextView.importantForAccessibility =
+        View.IMPORTANT_FOR_ACCESSIBILITY_NO
+    }
```

Comment:
@vrajdesai78 Suggest changing this entire logic like this:
```
  private fun updateChapterDescription() {
    val chapterDescription = htmlParserFactory.create(
      resourceBucketName,
      resumeLessonViewModel.entityType,
      explorationId,
      imageCenterAlign = true
    ).parseOppiaHtml(
      resumeLessonViewModel.chapterSummary.get()!!.summary,
      binding.resumeLessonChapterDescriptionTextView
    )
    if (chapterDescription.isNotBlank()) {
      binding.resumeLessonChapterDescriptionTextView.visibility = View.VISIBLE
      binding.resumeLessonChapterDescriptionTextView.text = chapterDescription
    } else {
      binding.resumeLessonChapterDescriptionTextView.visibility = View.GONE
    }
  }
```

**Why this change?**
In previous solution we are just controlling the accessibility-importance but if we enable `Show Layout Bounds` in `Developer Options` the `empty` textview is still available. But this solution gets rid of the `empty textview` completely which is much better solution.

I believe you can easily write test case for this solution?

Reply:
Yes, it sounds good to me. I have updated code based on your suggestion. Thanks @rt4914